pom.xml

```diff
@@ -21,13 +21,23 @@
         <dependency>
             <groupId>com.offbytwo.jenkins</groupId>
             <artifactId>jenkins-client</artifactId>
-            <version>0.3.6</version>
+            <version>0.3.7-SNAPSHOT</version>
```

Comment:
Do they have a CI running somewhere, where we can suck down the dependency ? 
Not a blocker - just asking